ml/ad_charts.cc

```diff
@@ -327,7 +327,7 @@ void ml_update_host_and_detection_rate_charts(ml_host_t *host, collected_number
                     "anomaly_detection", // family
                     "anomaly_detection.detector_events", // ctx
                     "Anomaly detection events", // title
-                    "percentage", // units
+                    "boolean", // units
```

Comment:
```suggestion
                    "status", // units
```

I think `status` is better, we usually use `status` for 1/0.

Reply:
agree will be less confusing to end user when they end up looking at it.